### DISCUSSION TITLE
查找hub可以用的模型

### DISCUSSION THREAD

**Discussion by tanglaoya321** [PaddlePaddle/PaddleFormers#2205] [Q&A] [answered]
哪里可以找到paddlehub可以使用的所有模型，readme 里的貌似不全，比如stable_diffution 就没有，还有就是看每个模型的大小

**Comment by rainyfly** [comment, marked as answer]
https://www.paddlepaddle.org.cn/hublist 这里列的是咱们PaddleHub目前所有的模型列表

**Reply by tanglaoya321** [reply]
感谢

**Reply by tanglaoya321** [reply]
这上面的模型的下载地址有吗，看到aistudio 里不少人直接使用了地址

**Reply by rainyfly** [reply]
使用 hub install 模型名字可以直接下载，使用这个命令下载的时候其实会显示出模型包的下载地址的

**Reply by tanglaoya321** [reply]
好的，感谢
